app/src/main/res/layout/fragment_tour_languages.xml

```diff
@@ -51,7 +45,7 @@
                 android:layout_marginStart="8dp"
                 android:layout_toEndOf="@id/action_switch_language"
                 android:layout_toRightOf="@id/action_switch_language"
```

Comment:
the layout of this button is now incorrect because you didn't remove layout attributes related to the removed button. Also, this leaves the `RelativeLayout` as useless, so you can remove it and put the remaining Button in it's place.

these attributes should be removed:
`layout_marginLeft`
`layout_marginStart`
`layout_toEndOf`
`layout_toRightOf`

Reply:
this is probably what triggered the fatal lint error

Reply:
Thanks for explaining!